lib/cisco_node_utils/cmd_ref/bridge_domain_vni.yaml

```diff
@@ -0,0 +1,28 @@
+# bridge_domain
+# bridge_domain feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k, ios_xr]
+
+create:
+  set_value: "bridge-domain <crbd>"
```

Comment:
Nit: I know variable naming is subjective but it's easier on the maintainers if you're consistent; i.e. `/crbd/bd/`